private-templates-service/client/src/components/Dashboard/PreviewModal/TemplateInfo/TemplateInfo.tsx

```diff
@@ -66,9 +66,10 @@ const cards = [
   }
 ];
 
-interface Props {
+interface Props extends RouteComponentProps<any> {
   template: Template;
   onSwitchVersion: (templateVersion: string) => void;
+  history: any;
```

Comment:
Lets try to avoid `any`.

History does not need to be included in `Props`

Reply:
Done.